src/Microsoft.AspNetCore.Razor.Language/RazorParserFeatureContext.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal abstract class RazorParserFeatureContext
+    {
+        internal static readonly Version LatestRazorVersion = new Version(2, 1, 0);
```

Comment:
Think about using an enum here instead. Version allows you to express a huge range of stuff including versions that done exist.

Think continuous vs discrete.

We never need/want to handle a version we didn't ship.